src/snowflake/cli/plugins/workspace/commands.py

```diff
@@ -33,3 +36,24 @@ def validate(
     """Validates the project definition file."""
     # If we get to this point, @with_project_definition() has already validated the PDF schema
     return MessageResult("Project definition is valid.")
+
+
+@ws.command(requires_connection=True)
+@with_project_definition()
+def bundle(
+    entity_id: str = typer.Option(
+        help=f"""The ID of the entity you want to bundle.""",
+    ),
+    **options,
+):
+    """
+    Prepares a local folder with the configured artifacts of the specified entity.
+    """
+
+    cli_context = get_cli_context()
+    ws = WorkspaceManager(
+        project_definition=cli_context.project_definition,
+        project_root=cli_context.project_root,
+    )
+
+    ws.bundle(entity_id)
```

Comment:
This is `ws bundle <entity_id>` right? I was convinced that we want to have `ws :entity_id bundle`?

Reply:
Yes eventually it's what we'll have, but it requires more infra to support this command syntax. Until it's ready I temporarily used the `--entity-id` argument syntax.